python/Ganga/Core/GangaThread/GangaThreadPool.py

```diff
@@ -93,6 +95,7 @@ def _really_shutdown(self, should_wait_cb=None):
         shutdown_thread.start()
 
         t_start = time.time()
+        t_last = -t_start
```

Comment:
This is actually part of the logic to check for the first time the user is prompted as there are two timers involved in that. It is a bit confusing though so Il'l add a comment 😄